package-lock.json

```diff
@@ -422,6 +418,7 @@
         "sprintf-js": "~1.0.2"
       }
     },
+>>>>>>> d3a281f... Refactor and clean
```

Comment:
this file has a number of merge conflict tags in it. could you clean it up a bit? thank you!

Reply:
done - interestingly enough it passes the tests with those conflicts.